client/www/components/dash/Storage.tsx

```diff
@@ -612,3 +661,65 @@ export function StorageTab({
     </div>
   );
 }
+
+function StorageDisabledTab({
+  className,
+  app,
+}: {
+  className?: string;
+  app: InstantApp;
+}) {
+  return (
+    <div className={cn('flex-1 flex flex-col', className)}>
+      <div className="flex-1 flex flex-col items-center justify-center">
+        <div className="text-center text-lg font-medium text-gray-900">
+          Storage is not enabled for this app yet!
+        </div>
+        <div className="text-center text-base text-gray-500">
+          If you'd like to try it out, please request beta access and we'll get
+          back to you ASAP.
+        </div>
+        {/* TODO: link to Google Form instead */}
+        <a
+          href="mailto:hello@instantdb.com"
+          target="_blank"
+          rel="noopener noreferer"
+        >
+          <Button className="mt-4" size="large">
+            Request beta access
+          </Button>
+        </a>
+      </div>
+    </div>
+  );
+}
```

Comment:
<img width="948" alt="Screenshot 2024-08-26 at 10 47 30 AM" src="https://github.com/user-attachments/assets/5374611d-7173-4f66-8925-e72c09dd702c">

Reply:
I like it!